drivers/overlay/overlay.go

```diff
@@ -181,7 +181,7 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		return nil, fmt.Errorf("Storage option overlay.size only supported for backingFS XFS. Found %v", backingFs)
 	}
 
-	logrus.Debugf("backingFs=%s, projectQuotaSupported=%v", backingFs, projectQuotaSupported)
+	logrus.Debugf("backingFs=%s, projectQuotaSupported=%v, useNativeDiff=%v", backingFs, projectQuotaSupported, !useNaiveDiff(home))
```

Comment:
I know not part of this change, but curious why the variable useNaiveDiff and useNaiveLock are Naive and not Native?

Reply:
`NaiveDiffDriver` is the name of the wrapper object that several of the storage drivers use to implement their `Changes()`/`Diff()`/`DiffSize()`/`ApplyDiff()` methods, so I expect that variables related to deciding whether or not its use is necessary are named after it.  The "native" implementation in the overlay driver uses knowledge of how the overlay filesystem works to gain speed advantages, but it can't cope with certain kernel features, so that decision has to be made at startup-time.

Reply:
Thanks nalind!